### PR TITLE
style(prettier): enforce default style in new TS files

### PR DESCRIPTION
**Pull request by Thomaash**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -102,5 +102,88 @@ module.exports = {
...
+        "test/edges/helpers.ts",
+        "test/edges/straight-edge.test.ts",
+        "test/gephi-parser.test.ts",
+        "test/helpers/index.ts"
+      ],
+      rules: {
+        "prettier/prettier": [
+          "error",
+          {
+            endOfLine: "lf",
```
Comment (mojoaxel) [inline .eslintrc.js]: @Thomaash I thought we decided to use the default prettier config?

File: .eslintrc.js [hunk 2]
```diff
@@ -102,5 +102,88 @@ module.exports = {
         'valid-typeof': 'off',
       },
     },
-  ],
-}
+    {
+      files: [
+        "lib/index-bundle.ts",
```
Comment (mojoaxel) [inline .eslintrc.js]: could we not use a glob like `lib/**/*.ts`?

File: .eslintrc.js [hunk 3]
```diff
@@ -102,5 +102,88 @@ module.exports = {
...
+            requireParamDescription: true,
+            requireParamType: false,
+            requireReturn: false, // Requires return for void functions.
+            requireReturnDescription: true,
+            requireReturnType: false
+          }
+        ],
+
+        // Class related.
+        "@typescript-eslint/member-naming": [
```
Comment (mojoaxel) [inline .eslintrc.js]: Once more: Why do we need eslint if we already have prettier?

Comment (Thomaash) [reply .eslintrc.js]: Prettier is only about formatting. ESLint does a lot more than that (it reports unused variables, missing JSDoc etc.)

File: .eslintrc.js (hunk 1 above)
Comment (Thomaash) [reply .eslintrc.js]: This is just for existing files. The default config will be enforced in new files.

File: .eslintrc.js (hunk 2 above)
Comment (Thomaash) [reply .eslintrc.js]: If we reformat the old files, then yes.

Comment (mojoaxel) [reply .eslintrc.js]: Ok, this should be cleaned up later, than.

File: .eslintrc.js (hunk 1 above)
Comment (mojoaxel) [reply .eslintrc.js]: I think we should leave th old *.js files alone and just format the new *.ts files.

Comment (Thomaash) [reply .eslintrc.js]: Eh, this is exactly what this config does.

**Pull request merged**